src/components/_channel/ChannelCardBase/ChannelCardBase.stories.tsx

```diff
@@ -22,12 +26,23 @@ export default {
   ],
 } as Meta
 
-const Template: Story<ChannelCardBaseProps> = (args) => {
-  return (
-    <>
-      <ChannelCardBase {...args} />
-    </>
-  )
-}
+const Template: Story<ChannelCardBaseProps> = (args) => <ChannelCardBase {...args} />
 
 export const Default = Template.bind({})
+export const WithButton = Template.bind({})
+WithButton.args = {
+  onFollow: () => null,
```

Comment:
Let's hide it from table
![Zrzut ekranu 2022-01-12 o 11 21 38](https://user-images.githubusercontent.com/13706246/149122186-7b89dbd1-d494-495c-ad8a-41e1562cb120.png)